connectors/es/sink.py

```diff
@@ -361,7 +374,7 @@ def __init__(
         self.total_docs_updated = 0
         self.total_docs_created = 0
         self.total_docs_deleted = 0
-        self.fetch_error = None
+        self.extractor_error = None
```

Comment:
Extractor used to be called Fetcher, so I assume `fetch_error` was a vestigial name. It doesn't make sense, so I think we should rename it. I'd also support it just being `self.error` - WDYT?

Reply:
`self.error` sounds good to me, we're already in `Extractor` class, so I don't see a need to prefix it with itself

Reply:
I think `self.error` makes more sense, we're already in the `Extractor` class, so I don't see a need to prefix the field with the class name

Reply:
Done.